### PR TITLE
feat(messaging): add reference content - sns/sqs overview

### PR DESCRIPTION
**Pull request by RoRoJ**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Add Messaging Reference Doc: SQS/SNS overview

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: compute/messaging/reference-content/snssqs-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  - messaging
+---
+
+## What is SNS/SQS?
+
+SNS and SQS are both managed messaging services developed by AWS and supported by Scaleway Messaging. Although they are two separate services, they are often discussed simultaneously since they are frequently coupled to work together. A Scaleway Messaging SNS/SQS namespace supports both services simultaneously.
+
+### SNS
+
+SNS (Simple Notification Service) was released in 2010. It enables users to interconnect their applications and systems using a [publish/subscribe](/compute/messaging/concepts#publishsubscribe) pattern. Many publishers can send messages to [topics](/compute/messaging/concepts#topicexchange), while consumers, e.g. applications, can subscribe to topics to receive all (or a subset) of messages sent to this topic. Scaleway Messaging makes it easy to migrate your SQS-connected applications by providing a dedicated protocol gateway.
```
Comment (JulienVdG) [inline compute/messaging/reference-content/snssqs-overview.mdx:19]:
```suggestion
SNS (Simple Notification Service) was released in 2010. It enables users to interconnect their applications and systems using a [publish/subscribe](/compute/messaging/concepts#publishsubscribe) pattern. Many publishers can send messages to [topics](/compute/messaging/concepts#topicexchange), while consumers, e.g. applications, can subscribe to topics to receive all (or a subset) of messages sent to this topic. Scaleway Messaging makes it easy to migrate your SNS-connected applications by providing a dedicated protocol gateway. 
```

File: compute/messaging/reference-content/snssqs-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+</Message>
+
+### SQS
+
+SQS (Simple Queue Service) became generally available in 2006. It is a highly scalable [queuing](/compute/messaging/concepts#queuing) service that facilitates the decoupling of software systems. Scaleway Messaging provides a gateway that enables you to connect your existing applications without having to significantly modify your code baseline.
+
+<Message type="note">
+
+  Some SNS features are not yet supported. Refer to the [developers documentation](https://developers.scaleway.com/en/products/messaging_and_queuing/api/v1alpha1/) for more information.
```
Comment (JulienVdG) [inline compute/messaging/reference-content/snssqs-overview.mdx:39]:
```suggestion
  Some SQS features are not yet supported. Refer to the [developers documentation](https://developers.scaleway.com/en/products/messaging_and_queuing/api/v1alpha1/) for more information.
```

**Review by JulienVdG — COMMENTED** [review]
Just spotted SNS vs SQS typos

File: compute/messaging/reference-content/snssqs-overview.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Credentials are scoped to the namespace (the bearer of the credentials can access this Messaging namespace only). You can choose to give each set of credentials the required level of permissions within the namespace:
+
+  - **Read**: receive messages from queues and topics.
+  - **Write**: send messages to queues and publish in topics.
+  - **Manage**: create or edit resources (such as queues and topics).
+
+## Further actions
+
+All further actions including creating, listing and managing queues and topics, sending messages etc. can be done using the AWS CLI or AWS SDKs. Check out our documentation to help you get started with the[AWS CLI](/compute/messaging/api-cli/connect-aws-cli/) or [AWS SDKs](/compute/messaging/api-cli/python-node-sqs/).
```
Comment (ldecarvalho-doc) [inline compute/messaging/reference-content/snssqs-overview.mdx:59]:
```suggestion
All further actions including creating, listing and managing queues and topics, sending messages etc. can be done using the AWS CLI or AWS SDKs. Check out our documentation to help you get started with the [AWS CLI](/compute/messaging/api-cli/connect-aws-cli/) or [AWS SDKs](/compute/messaging/api-cli/python-node-sqs/).
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by lucasscw — APPROVED** [review]

**Pull request merged**